example/flaskr/cookbook.py

```diff
@@ -0,0 +1,56 @@
+from flask import Blueprint
+from flask import flash
+from flask import g
+from flask import redirect
+from flask import render_template
+from flask import request
+from flask import url_for
+from werkzeug.exceptions import abort
+
+from flaskr.auth import login_required
+from flaskr.db import get_db
+from flaskr.ingredient import getAlternativesByRatingAvg
+from flaskr.recipe import get_recipe, get_recipe_ingredients
+import time
+import sqlite3
+
+bp = Blueprint("cookbook", __name__, url_prefix="/cookbook")
+
+
+@bp.route("/add_recipe/<int:recipe_id>", methods=("GET",))
```

Comment:
Not totally happy with this being a GET route, but it seems to be the easiest way to do it given that we just have an `<a>` tag on the front end. If we wanted to change it to POST, we'd have to add a form or some `onclick` Javascript.